rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactorySpec.groovy

```diff
@@ -0,0 +1,44 @@
+package com.netflix.spinnaker.rosco.providers.util
+
+import spock.lang.Shared
+import spock.lang.Specification
+
+class LocalJobFriendlyPackerCommandFactorySpec extends Specification {
+
+    @Shared
+    LocalJobFriendlyPackerCommandFactory packerCommandFactory = new LocalJobFriendlyPackerCommandFactory()
+
```

Comment:
Can you please format this class like the other test classes? (2-space indentation, `setup/when/then` content indented, trim some of the extraneous blank lines)